config/mv_checks.xml

```diff
@@ -381,7 +383,7 @@
         <module name="OuterTypeFilename"/>
         <module name="TodoComment">
             <property name="format" value="TODO"/>
-            <property name="severity" value="warning"/>
+            <property name="severity" value="info"/>
```

Comment:
Can you add `<!-- TODO remove after MV5 -->` or something?

Reply:
hehe todo a toto